src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -3601,3 +3601,140 @@ def test_sql_instance_failover_group_mgmt(self):
+                 ' --retention-days {} --email-addresses {} --disabled-alerts {}'
+                 ' --email-account-admins {}'
+                 .format(resource_group, server, database_name, state_enabled, key,
+                         storage_endpoint, retention_days, email_addresses_input,
+                         disabled_alerts_input, email_account_admins),
+                 checks=[
+                     JMESPathCheck('resourceGroup', resource_group),
+                     JMESPathCheck('state', state_enabled),
+                     JMESPathCheck('storageAccountAccessKey', key),
+                     JMESPathCheck('storageEndpoint', storage_endpoint),
+                     JMESPathCheck('retentionDays', retention_days),
+                     JMESPathCheck('emailAddresses', email_addresses_expected),
+                     JMESPathCheck('disabledAlerts', disabled_alerts_expected),
+                     JMESPathCheck('emailAccountAdmins', email_account_admins)])
+
+        # list recommended sensitivity classifications
+        expected_recommended_sensitivityclassifications_count = 15
+        self.cmd('sql db classification recommendation list -g {} -s {} -n {}'
+                 .format(resource_group, server, database_name),
+                 checks=[
+                     JMESPathCheck('length(@)', expected_recommended_sensitivityclassifications_count)])
+
+        schema_name = 'SalesLT'
+        table_name = 'Customer'
+        column_name = 'FirstName'
+
+        # disable the recommendation for SalesLT/Customer/FirstName
+        self.cmd('sql db classification recommendation disable -g {} -s {} -n {} --schema-name {} --table-name {} --column-name {}'
+                 .format(resource_group, server, database_name, schema_name, table_name, column_name))
+
+        # list recommended sensitivity classifications
+        self.cmd('sql db classification recommendation list -g {} -s {} -n {}'
+                 .format(resource_group, server, database_name),
+                 checks=[
+                     JMESPathCheck('length(@)', expected_recommended_sensitivityclassifications_count - 1)])
+
+        # re-enable the disabled recommendation
+        self.cmd('sql db classification recommendation enable -g {} -s {} -n {} --schema-name {} --table-name {} --column-name {}'
+                 .format(resource_group, server, database_name, schema_name, table_name, column_name))
+
+        # lits recommended sensitivity classifications
+        self.cmd('sql db classification recommendation list -g {} -s {} -n {}'
+                 .format(resource_group, server, database_name),
+                 checks=[
+                     JMESPathCheck('length(@)', expected_recommended_sensitivityclassifications_count)])
+
+        # update the sensitivity classification
+        information_type = 'Name'
+        label_name = 'Confidential - GDPR'
+        information_type_id = '57845286-7598-22f5-9659-15b24aeb125e'
```

Comment:
Where do these guids come from?

Reply:
Is there online help where I can find the builtin guids?

Reply:
There are default values sets for built-in labels, user can also specify custom guids